great-idea/css/index.css

```
@@ -63,4 +63,214 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+h1 {
+	line-height: 65px;
+  }
+
+  h2 {
+  font-weight: bold;
+  }
+  
+  a {
+	color: inherit;
+	text-decoration: none;
+  }
+  
+  .container {
+	width: 1024px;
+	min-height: 1px;
+	margin: auto;
+  }
+  
+  header {
+	margin-top: 27px;
+  }
+  
+  nav {
+	display: inline-block;
+	margin-left: 75px;
+	vertical-align: top;
+	margin-top: 10px;
+  }
+  
+  nav a {
+	color: rgb(188,188,188);
+	font-size: 14px;
+  }
+  
+  #nav-product {
+	margin-left: 50px;
+  }
+  
+  #nav-vision {
+	margin-left: 50px;
+  }
+  
+  #nav-features {
+	margin-left: 50px;
+  }
+  
+  #nav-about {
+	margin-left: 50px;
+  }
+  
+  #nav-contact {
+	margin-left: 50px;
+  }
+  
+  .logo {
+	vertical-align: right;
+	margin-left: 135px;
+  }
+  
```

```suggestion
  img#logo {
    margin-left: 75px;
}
```
header image is placed too closely to nav links.